modules/document_repository/jsx/docIndex.js

```diff
@@ -363,6 +364,7 @@ class DocIndex extends React.Component {
           label="Filter globally"
           id="globalSelection"
           value={this.state.global}
+           offset=''
```

Comment:
Spacing looks off here
```suggestion
          value={this.state.global}
          offset=''
```